### PR TITLE
Show metric description in Enterprise Console

### PR DESCRIPTION
**Pull request by jingcheng16**
## Product Description

Modified the Enterprise Console’s user interface to include descriptions for each metric.

image

## Technical Summary

Ticket: https://dimagi.atlassian.net/browse/SAAS-16309

Review by commit

## Feature Flag

No FF.

### Safety story

I have tested my changes locally
This PR is mainly to add a description in each tile.

### QA Plan

No QA

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/enterprise/enterprise.py [hunk 1]
```diff
@@ -115,7 +123,13 @@ def total(self):
 
 
 class EnterpriseDomainReport(EnterpriseReport):
-    title = _('Project Spaces')
```
Comment (biyeun) [inline corehq/apps/enterprise/enterprise.py:118]: @jingcheng16 you can just leave it as a class property. no need to do 
``
@property
def title(self):
    ...
```
etc...
```
title = ....
```
works just fine with the `ABC` class. can you make this update for readability?

Comment (jingcheng16) [reply corehq/apps/enterprise/enterprise.py:118]: a127b2699679c7e5879737c851bc1a08c350da14

File: corehq/apps/enterprise/enterprise.py [hunk 2]
```diff
@@ -40,8 +41,15 @@ class EnterpriseReport:
...
-    title = _('Enterprise Report')
-    subtitle = ''
+    @property
+    @abstractmethod
+    def title(self):
+        pass
+
+    @property
+    @abstractmethod
+    def description(self):
```
Comment (mjriley) [inline corehq/apps/enterprise/enterprise.py:51]: Something feels off about the enterprise report containing a `description`. I feel like this isn't really a property of the report, but rather the 'view model'. But given how the code is now structured, I'm not sure I'd recommend ripping out this property and creating separate view models. How would you feel about renaming it to like `summary_description`, as it seems to describe the summary information more than the entire report

**Review by mjriley — APPROVED** [review]
Non-blocking comment

File: corehq/apps/enterprise/enterprise.py (hunk 2 above)
Comment (jingcheng16) [reply corehq/apps/enterprise/enterprise.py:51]: Or maybe `total_description`, given the report has a total property?

Comment (mjriley) [reply corehq/apps/enterprise/enterprise.py:51]: Sure, I think either of those work, although `total_description` is probably more straight-forward. `description_of_total` sounds a little odd

Comment (jingcheng16) [reply corehq/apps/enterprise/enterprise.py]: 55df4bd451cfa66b148d72b28fda998a978bf589

**Review by mjriley — APPROVED** [review]

**Pull request merged**